channels/stable-4.16.yaml

```diff
@@ -1,9 +1,49 @@
 feeder:
   delay: PT0H
-  filter: 4[.](15|16)[.][0-9].*
+  filter: 4[.](14|15|16)[.][0-9].*
   name: stable
 name: stable-4.16
 versions:
+- 4.14.0
```

Comment:
Excluding `fast-4.16` here is [causing][1]:

```
Error: Error in channel ordering: ["Release 4.14.30 present in stable-4.16, but not in fast-4.16...
```

Probably easier to do the same thing to `fast-4.16` and `candidate-4.16`, but I'll leave those out for now, while we wait to collect feedback, in case folks think it's worth just doing `stable-4.16` (with a CI `/override ...`?) for some reason.

[1]: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_cincinnati-graph-data/6268/pull-ci-openshift-cincinnati-graph-data-master-e2e/1859293525741408256#1:build-log.txt%3A25

Reply:
Yeah I've assumed we'd just make all channels have effectively the same filters as eus channels modulo any rollouts in progress (ie 4.18).